### PR TITLE
Add description to script

### PR DESCRIPTION
**Pull request by mmlb**
Just some metadata that can be useful in various contexts like enterShell for example.

Closes: #887

### CODE REVIEW

**Comment by mmlb** [comment]
:wave: @domenkozar, just wanted to bring this to your attention.

File: docs/scripts.md [hunk 1]
```diff
@@ -57,3 +57,26 @@ Entering shell ...
...
 }
 ```
+
+Scripts can also have an optional description, which can be used in your `enterShell`.
+
+```nix title="devenv.nix"
+{ pkgs, ... }:
+
+{
+  scripts.silly-example.exec = ''
```
[the remark this answers is not in the document]
Comment (mmlb) [reply docs/scripts.md:67]: yep, typo :(. Fixed in latest push, thanks for taking a look.

File: docs/scripts.md [hunk 2]
```diff
@@ -57,3 +57,26 @@ Entering shell ...
...
+      echo 🦾 Helper scripts you can run to make your development richer:
+      echo 🦾
+      sed -e 's| |••|g' -e 's|=| |' <<EOF | column -t | sed -e 's|^|🦾 |' -e 's|••| |g'
+      ${lib.generators.toKeyValue {} (lib.mapAttrs (name: value: value.description) config.scripts)}
+      EOF
+      echo
+    '';
+}
+```
+
```
Comment (domenkozar) [inline docs/scripts.md:82]: Let's also show example output.

Comment (mmlb) [reply docs/scripts.md:86]: Done. I've also added some more info in the examples section too.

**Comment by mmlb** [comment]
awesome thanks!

**Pull request merged**